cli/smartcontract/smart_contract.go

```diff
@@ -98,30 +106,46 @@ func NewCommands() []*cli.Command {
 	invokeFunctionFlags = append(invokeFunctionFlags, options.RPC...)
 	deployFlags := append(invokeFunctionFlags, []cli.Flag{
 		&cli.StringFlag{
-			Name:    "in",
-			Aliases: []string{"i"},
-			Usage:   "Input file for the smart contract (*.nef)",
+			Name:     "in",
+			Aliases:  []string{"i"},
+			Required: true,
+			Usage:    "Input file for the smart contract (*.nef)",
+			Action: func(c *cli.Context, value string) error {
+				if value == "" {
+					return cli.Exit(fmt.Errorf("the i flag requires a non-empty value"), 1)
+				}
+				return nil
```

Comment:
Move it to some helper. Provide flag name as an argument to this helper. Reuse this helper from every Action for required flags.